cmd/promtool/main.go

```diff
@@ -434,11 +436,22 @@ func QueryInstant(url, query string, p printer) int {
 		return 1
 	}
 
+	var eTime time.Time
+	if evalTime == "" {
```

Comment:
Just start with `eTime = time.Now()` and if `evalTime != ""` parse? (: Might be bit simpler!

Reply:
done! thx